virttest/qemu_vm.py

```diff
@@ -457,7 +457,8 @@ def add_qmp_monitor(devices, monitor_name, filename):
         def add_serial(devices, name, filename):
             if (not devices.has_option("chardev") or
                     not any(devices.has_device(dev)
-                            for dev in ("isa-serial", "sclpconsole", "spapr-vty"))):
+                            for dev in ("isa-serial", "sclpconsole", "spapr-vty")) or
+                    'aarch64' in params.get('vm_arch_name', arch.ARCH)):
```

Comment:
This is addressed in https://github.com/avocado-framework/avocado-vt/pull/1684/commits/3d8084ade9aac8f3f76885642dbb7dbd6e0fdd78 which is older (2 months) with additional fixes related to s390x so it'd be probably useful to use that, what do you think?

Reply:
No problem. I'll remove it.